src/fparser/common/utils.py

```diff
@@ -331,7 +333,7 @@ def __getattr__(self, name):
             raise AttributeError('instance does not have attribute %r' % (name))
         return cls
 
-class classes(with_metaclass(meta_classes, type)):
+class classes(six.with_metaclass(meta_classes, type)):
```

Comment:
Also pylint:C:336, 0: Class name "classes" doesn't conform to PascalCase naming style (invalid-name) but I'm not sure what this class does so if it is better to leave it as it is then please ignore the error.

Reply:
Ignoring.